app/views/course/users/_user_list_data.json.jbuilder

```diff
@@ -7,7 +7,11 @@ json.name course_user.name.strip
 json.imageUrl user_image(course_user.user)
 json.email course_user.user.email
 
+reference_timeline_id = course_user.reference_timeline_id
```

Comment:
abstract as `reference_timeline_for(course_user)` and use it for `reference_timeline_id` in the `/coursemology2/app/models/course/lesson_plan/item.rb` file line 138